src/realmGeol.ttl

```diff
@@ -57,307 +289,1801 @@ sorelsp:hasUpperBoundary rdf:type owl:ObjectProperty .
+                                             <https://orcid.org/0000-0001-5489-9590> ,
+                                             <https://orcid.org/0000-0002-8742-7730> ;
+                         dcterms:source <https://doi.org/10.1306/05230505015> ;
+                         skos:definition "A Lithodemic Unit is a defined body of predominantly intrusive, highly metamorphosed, or intensely deformed rock that, because it is intrusive or has lost primary structure through metamorphism or tectonism, generally does not conform to the Law of Superposition. Expressions of age of lithodemic units (because they are non-stratigraphic) should be given in terms of the appropriate geochronologic (geological-time) unit, and thus not by chronstratigraphic (time-rock) terms. (North American Stratigraphic Code, 2005)"@en ;
+                         skos:prefLabel "lithodemic feature"@en .
 
 
 ###  http://sweetontology.net/realmGeol/Lithosphere
 soreag:Lithosphere rdf:type owl:Class ;
-                 rdfs:subClassOf soreag:GeosphereLayer ,
-                                 [ rdf:type owl:Restriction ;
-                                   owl:onProperty sorelsc:containsMaterial ;
-                                   owl:allValuesFrom somarock:Rock
-                                 ] ;
-                 rdfs:label "lithosphere"@en .
+                   rdfs:subClassOf soreag:GeosphereLayer ,
+                                   [ rdf:type owl:Restriction ;
+                                     owl:onProperty sorelsc:containsMaterial ;
+                                     owl:allValuesFrom somarock:Rock
+                                   ] ;
+                   rdfs:label "lithosphere"@en .
+
+
+###  http://sweetontology.net/realmGeol/LithostratigraphicUnit
+soreag:LithostratigraphicUnit rdf:type owl:Class ;
+                              rdfs:subClassOf soreag:StratigraphicUnit ;
+                              dcterms:contributor <http://linked.data.gov.au/org/gsq> ,
+                                                  <https://orcid.org/0000-0001-5489-9590> ,
+                                                  <https://orcid.org/0000-0002-8742-7730> ;
+                              dcterms:source <http://www.stratigraphy.org/index.php/ics-stratigraphicguide> ;
+                              skos:definition "Lithostratigraphic units are bodies of rocks, bedded or unbedded, that are defined and characterized on the basis of their lithologic properties and their stratigraphic relations. Lithostratigraphic units are the basic units of geologic mapping. (ICS, Online, December 2019)"@en ;
+                              skos:prefLabel "lithostratigraphic unit"@en .
 
 
 ###  http://sweetontology.net/realmGeol/Lopolith
 soreag:Lopolith rdf:type owl:Class ;
-              rdfs:subClassOf soreag:PlutonicStructure ;
-              rdfs:label "lopolith"@en .
+                rdfs:subClassOf soreag:PlutonicStructure ;
+                rdfs:labe
```

Comment:
Spelling? Usually **Tec~h~tonostratigraphic**

Reply:
I think the "tech" is always me! Likely it's a smart spell checker that knows I often write about tech!